sdk/ml/azure-ai-ml/azure/ai/ml/operations/_index_operations.py

```diff
@@ -224,3 +250,236 @@ def cls(rest_indexes: Iterable[RestIndex]) -> List[Index]:
             return self._azure_ai_assets.indexes.list_latest(cls=cls)
 
         return self._azure_ai_assets.indexes.list(name, list_view_type=list_view_type, cls=cls)
+
+    # pylint: disable=too-many-locals
+    def build_index(
```

Comment:
Is it worth to add @experimental annotation here? @kdestin I know you had concern on printing too many warning messages. What do you think?

Reply:
I will follow up with @kdestin and address this in the next PR

Reply:
We currently have `@experimental` on:

* The index operations `ml_client.indexes`
* The model `Index`
* `load_index`

We were told in api review that it's probably not worth annotating individual methods